<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lienkettkmoi_form</class>
 <widget class="QMainWindow" name="lienkettkmoi_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liên kết tài khoản mới</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 243, 253);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);
font: 16pt &quot;Cambria&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Liên kết tài khoản mới&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 205, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/left-arrow.png</normaloff>../../../Downloads/left-arrow.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 234, 255);
font: 12pt &quot;Cambria&quot;;
border-radius:5px</string>
    </property>
    <property name="text">
     <string>  Thông tin liên kết</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>87</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tên ngân hàng:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_timnh">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Tên ngân hàng</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Số thẻ / tài khoản:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_stk">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Số thẻ / tài khoản</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tên chủ thẻ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_ten">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Tên chủ thẻ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_huy">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);
</string>
    </property>
    <property name="text">
     <string>Hủy bỏ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lienket">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 209);
</string>
    </property>
    <property name="text">
     <string>Liên kết ngay</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
